libraries/AP_Scripting/lua_scripts.cpp

```diff
@@ -40,6 +40,15 @@ uint32_t lua_scripts::loaded_checksum;
 uint32_t lua_scripts::running_checksum;
 HAL_Semaphore lua_scripts::crc_sem;
 
+// return string error message for error object at top of stack
+static const char *get_error_object_message(lua_State *L) {
+    const char *m = lua_tostring(L, -1);
+    if (!m) { // error object is not stringifiable
+        return "<error>";
```

Comment:
Should this be (char *)"<error>" - I'm pretty sure this won't compile on my Mac as is.

Reply:
The macOS SITL CI build passed, and the `char[]` literal type will decay to a const pointer just fine?

Reply:
Yeah, I think its fine.